python/ray/workflow/execution.py

```diff
@@ -61,22 +61,26 @@ def run(entry_workflow: Workflow,
         except Exception:
             wf_exists = False
 
+        # "Is growing" means we could adding steps to the (top-level)
+        # workflow to grow the workflow dynamically at runtime.
+        is_growing = (step_type not in (StepType.FUNCTION, StepType.WAIT))
```

Comment:
should it be: is virtual actor and just check virtual actor type?

Reply:
I think this is complementary to checking the virtual actor type. It have 2 types too. This one is more close to the original implementation where `StepType.FUNCTION` is used.